lib/resources/r4/charge_item_create.yaml

```diff
@@ -291,7 +291,7 @@ fields:
       <ul>
         <li>Custom-attribute-name and Custom-attribute-value are both required</li>
         <li>id and valueString for custom-attribute-value are required</li>
-        <li>Custom-attribute-value can be of type integer, string or decimal</li>
+        <li>Custom-attribute-value can be of type integer, string, or decimal</li>
```

Comment:
Do we need to list dateTime as a valid value type for charge-item-create custom attributes? I wasn't sure.

Reply:
Yes please. It should be the same as modify.

Reply:
updated - https://github.com/cerner/fhir.cerner.com/pull/799/commits/09e653a93d3e2fc66fe9587756147d628d951378

Reply:
Thanks!